db/model/model.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../../db/connection");
+
+exports.getTopic = () => {
```

Comment:
looks like you're submitting two different models. Please only submit code relevant to the ticket you're working on